testing/tests/DevExpress.ui.widgets.editors/lookup.tests.js

```diff
@@ -1594,6 +1596,25 @@ QUnit.module('options', {
         assert.equal(autoValue, initialValue, 'initial value equal auto value');
     });
 
+    QUnit.test('popup height should have correct size on mobile devices', function(assert) {
+        if(devices.real().deviceType !== 'phone') {
+            assert.ok(true, 'not mobile device');
+            return;
+        }
+        const initialVisualViewport = window.visualViewport;
+        window.visualViewport = { height: 510, width: 405 };
```

Comment:
You should use try {} finally {} construction for it and for setting initial visualViewport.